app/controllers/games_controller.rb

```diff
@@ -0,0 +1,28 @@
+class GamesController < ApplicationController
+	# before_action :authenticate_user!
+  
+  def show
+  	@game = Game.find(params[:id])
+  end
+
+  def new
+  	@game = Game.new
+  end
+
+  def create
+  	@game = Game.new(game_params)
+  	if @game.save
+  		redirect_to game_path(@game)
+  	else
+  		render :new, :status => :unprocessable_entity 
+  	end
+
+  end
+
+  private
+
+  def game_params
+		params.require(:game).permit(:name)  	
+  end
+
+end
```

Comment:
Thanks for the instructions! I applied them to the file on my machine and pushed them up, but it doesn't appear the change worked. I'm probably doing a step wrong.